test/sanity/shellcheck/skip.txt

```diff
@@ -1,2 +1,3 @@
 hacking/unify_repos.sh
 test/sanity/code-smell/boilerplate.sh
+test/sanity/code-smell/no-smart-quotes.sh
```

Comment:
Instead of skipping the entire file, you can use inline comments to skip specific tests on specific lines:

https://github.com/koalaman/shellcheck/wiki/Ignore